docs/beta/tutorial/get_version.md

```diff
@@ -0,0 +1,51 @@
+# Tutorial: Get Tamr version
+This tutorial will introduce basic usage of the Tamr Python client.  It will guide you through the process of configuring the connection to a Tamr instance and retrieving the version of that instance.
+
+## Preparation
+To complete this tutorial you will need access to a Tamr instance.  Specifically you should have a username and password that allow you to log in to Tamr, as well as the socket address of the instance.  The socket address is composed of
+1. The protocol, such as `"https"` or `"http"`
+2. The host, which may be `"localhost"` if the instance is deployed from the same machine from which your Python code will be run
+3. The port at which you access the Tamr user interface, typically `9100`
+
+When you view the Tamr user interface in a browser, the url is `<protocol>://<host>:<port>`.
+
+## Procedure
+### The Session Object
+The Tamr Python client uses `Session` objects primarily to persist the user's authentication details across requests made to the server where Tamr is hosted.
+
+A `Session` object carries authentication credentials derived from a username and password, and is not explicitly tied to any single Tamr instance. For more details, see the documentation for the [Requests library](https://requests.readthedocs.io/en/master/user/advanced/#session-objects).
```

Comment:
The word "object" carries no semantic meaning, so I suggest removing it:

```suggestion
### The Session
The Tamr Python client uses `Session`s primarily to persist the user's authentication details across requests made to the server where Tamr is hosted.

A `Session` carries authentication credentials derived from a username and password, and is not explicitly tied to any single Tamr instance. For more details, see the documentation for the [Requests library](https://requests.readthedocs.io/en/master/user/advanced/#session-objects).
```

This goes for other parts of the tutorial (e.g. "instance object" -> "instance").

(I know the requests library calls them "session objects", but let's not repeat that mistake 😉 )